main/k8s.tf

```diff
@@ -0,0 +1,75 @@
+locals {
+    labels = {
+        "app.kubernetes.io/name" = var.k8s_app_name
+        "app.kubernetes.io/instance" = var.k8s_app_name
+        "app.kubernetes.io/managed-by" = var.k8s_app_name
+    }
+}
+
+resource "kubernetes_config_map" "beekeeper" {
+    count = var.instance_type == "k8s" ? 1 : 0
+    metadata {
+        name = var.k8s_app_name
+    }
+
+    data = {
+        "${local.scheduler_name}.properties" = data.template_file.beekeeper_path_scheduler_config
+        "${local.cleaner_name}.properties" = data.template_file.beekeeper_cleanup_config
+    }
+}
+
+resource "kubernetes_secret" "beekeeper" {
```

Comment:
I don't think we need the key to the secrets manager password to be a kubernetes secret, this can probably just go straight in as an environment variable. At some point one upgrade for Beekeeper could be to add a kubernetes password strategy.